packages/zowe-explorer/package.json

```diff
@@ -935,16 +935,6 @@
           "command": "zowe.uss.pasteUssFile",
           "group": "001_zowe_ussCreate@5"
         },
-        {
-          "when": "view == zowe.uss.explorer && viewItem =~ /^textFile.*/",
-          "command": "zowe.uss.binary",
-          "group": "002_zowe_ussSystemSpecific@1"
-        },
-        {
-          "when": "view == zowe.uss.explorer && viewItem =~ /^binaryFile.*/",
-          "command": "zowe.uss.text",
-          "group": "002_zowe_ussSystemSpecific@2"
-        },
```

Comment:
@t1m0thyj I see these right-click actions being removed from the USS menu.  Is there a new way to achieve this functionality in place that I am missing?  I have used this functionality a lot over the years and would really miss having it.

Reply:
nm, see PR that introduces the new functionality